src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -71,49 +71,47 @@
                 new PostalCode("540236"),
-                new Deadline(LocalDate.parse("01-10-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
-                new Remark("3 bedrooms"),
-                    getTagSet("3 bedrooms", "Renovation needed"))
+                new Deadline(LocalDate.parse("01-2-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Remark("Big living room"),
+                getTagSet("3 bedrooms", "Renovation needed"))
         };
     }
 
     public static Appointment[] getSampleAppointments() {
         return new Appointment[]{
             new Appointment(new Name("Meet Alex"),
-                new Remark("At M Hotel"),
+                new Remark("To celebrate Christmas at Fullerton Hotel"),
                 new Date(LocalDate.parse("25-12-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
                 new Time(LocalTime.parse("1500", DateTimeFormat.INPUT_TIME_FORMAT))),
             new Appointment(new Name("Meet Bob"),
-                new Remark("At Plaza Singapore Starbucks"),
-                new Date(LocalDate.parse("01-02-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
-                new Time(LocalTime.parse("2000", DateTimeFormat.INPUT_TIME_FORMAT))),
-            new Appointment(new Name("Meet Alice"),
-                new Remark("At client's house"),
-                new Date(LocalDate.parse("17-08-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
-                new Time(LocalTime.parse("1500", DateTimeFormat.INPUT_TIME_FORMAT))),
+                new Remark("At his house"),
+                new Date(LocalDate.parse("30-04-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Time(LocalTime.parse("1030", DateTimeFormat.INPUT_TIME_FORMAT))),
             new Appointment(new Name("Meet Caleb"),
-                new Remark("At void deck of his house"),
+                new Remark("At Causeway point Mcdonalds"),
                 new Date(LocalDate.parse("07-03-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
                 new Time(LocalTime.parse("1030", DateTimeFormat.INPUT_TIME_FORMAT))),
+            new Appointment(new Name("Meet Darren"),
+                new Remark("For signing of sales agreement at HDB"),
+                new Date(LocalDate.parse("12-10-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Time(LocalTime.parse("1400", DateTimeFormat.INPUT_TIME_FORMAT))),
+            new Appointment(new Name("Meet Emily"),
+                new Remark("At MayFair Gardens"),
+                new Date(LocalDate.parse("15-06-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Time(LocalTime.parse("1100", DateTimeFormat.INPUT_TIME_FORMAT))),
+            new Appointment(new Name("Meet Freddy"),
+                new Remark("At Orchard Centrepoint's Starbucks"),
+                new Date(LocalDate.parse("17-08-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Time(LocalTime.parse("1930", DateTimeFormat.INPUT_TIME_FORMAT))),
```

Comment:
should `AppointmentBuilder` class be used for this?

Reply:
Same as above